### DISCUSSION TITLE
[Idea] 一些奇特的想法

### DISCUSSION THREAD

**Discussion by shenmadouyaowen** [lobehub/lobehub#768] [Ideas]
### 🥰 需求描述 | Feature Description

主要用于一些模型不支持函数
是否可以做到一个模型做识别函数，然后调用函数获取信息，一个模型根据调用函数信息反馈给客户。

### 🧐 解决方案 | Proposed Solution

调用模型获取函数并读取信息，然后反馈给另外一个模型返回给用户

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
这个思路业界已经不少人在做了。相当于先进行「意图识别」，然后再「工具调用」，最后用 LLM 总结。这每一步都可以用不同的模型。

不过在 LobeChat 里不太好这么做，实现成本太高，而且环节一长，错误的可能性也会增多
